test_suite.py

```diff
@@ -18,6 +18,20 @@ def expected_read_file_output():
     return expected
 
 
+@dataclass
+class Python:
+    name="python"
+
+    def script(self, test_name):
+        return f'python {test_name}.py'
+
+# List of language classes with which to parametrize tests
+LANGUAGES = [Python(),]
+
+@pytest.fixture(params=LANGUAGES, ids=[x.name for x in LANGUAGES])
```

Comment:
Using `ids` to label each test by the name of the language

Reply:
I like that. Didn't know you could do that.